NLog.Web.AspNetCore/LayoutRenderers/AspNetRequestPostedBody.cs

```diff
@@ -125,25 +161,32 @@ bool TryEnableBuffering(HttpRequest httpRequest, long? contentLength, out Stream
                 return false;
             }
 
-            int bufferThreshold = MaxContentLength <= 0 ? 64 * 1024 : MaxContentLength;
+            int bufferThreshold = MaxContentLength <= 0 ? Size64KiloBytes : MaxContentLength;
             if (MaxContentLength == 0 && contentLength > bufferThreshold)
             {
                 InternalLogger.Debug("AspNetRequestPostedBody: body stream cannot seek and stream is too big. ContentLength={0}", contentLength);
                 return false;
             }
 
-#if ASP_NET_CORE2
-            Microsoft.AspNetCore.Http.HttpRequestRewindExtensions.EnableBuffering(httpRequest, bufferThreshold);
-            bodyStream = httpRequest.Body;
-            return bodyStream?.CanSeek == true;
-#elif ASP_NET_CORE1
-            Microsoft.AspNetCore.Http.Internal.BufferingHelper.EnableRewind(httpRequest, bufferThreshold);
+            EnableRewind(httpRequest, bufferThreshold);
+
+#if ASP_NET_CORE1 || ASP_NET_CORE2
             bodyStream = httpRequest.Body;
```

Comment:
I think this statement should be outside this method - do you agree? It's already in a later phase

Reply:
Would think that the EnableRewind would return the new buffered Stream.

Reply:
refactored. I'm pretty sure there is no need to get the Body property of the httprequest after enableRewind. So removed that. See latest commit